pyo3-ffi/src/object.rs

```diff
@@ -123,38 +131,45 @@ pub struct PyVarObject {
     pub ob_size: Py_ssize_t,
 }
 
-// skipped _PyVarObject_CAST
+// skipped private _PyVarObject_CAST
 
 #[inline]
 pub unsafe fn Py_Is(x: *mut PyObject, y: *mut PyObject) -> c_int {
     (x == y).into()
```

Comment:
Totally unrelated to this PR and a bit of a minor corner case, but this is wrong on pypy, see https://github.com/pypy/pypy/issues/4044

Reply:
Thanks for flagging, worth fixing anyway while I'm here!